lua/possession/display.lua

```diff
@@ -99,6 +99,7 @@ end
 ---@field buffers? boolean include buffers from parsed vimscript
 ---@field buffers_short? boolean show buffer paths normalized and relative to session cwd
 ---@field tab_cwd? boolean include tab cwds from parsed vimscript
+---@field only_cwd? boolean only show sessions for the cwd
```

Comment:
I think `display.echo_sessions` shouldn't take filtering options. Instead, pass the already filtered sessions in `opts.sessions`.

Reply:
Change made.